src/main/scala/bot/MessageFormatter.scala

```diff
@@ -12,12 +12,14 @@ class MessageFormatter {
       case (_, firstName, _) => firstName
     }
     s"""
-      |*$name*, welcome to Cavisto bot.
-      |
-      |_Available commands:_
-      |/advise -- Ask for wine advice, using filters
-      |/search -- Search wines by name. Example: /search cabernet
-      |""".stripMargin
+       |*$name*, welcome to Cavisto bot.
+       |
+       |_Available commands:_
+       |/start — Get a help message.
+       |/advise — Ask for wine advice, using filters.
+       |/search ... — Search wines by name. Example: /search cabernet
+       |/note — Create a note about wine.
```

Comment:
Would be interesting to see the commands available as an enum with descriptions which you could iterate through in here.